channel/handlers.go

```diff
@@ -2979,3 +2993,167 @@ func (server Server) handleChatEvent(conn mnet.Server, reader mpacket.Reader) {
 		log.Println("Unknown chat event type:", op)
 	}
 }
+
+func (d *player) removeAllCooldowns() {
+	if d == nil || d.skills == nil {
+		return
+	}
+	for id, ps := range d.skills {
```

Comment:
If you want to ignore id value then this can be re-written as `for _, ps := range d.skills {`